Source/WebCore/platform/ios/wak/WAKAppKitStubs.h

```diff
@@ -151,18 +150,21 @@ typedef NSUInteger NSRectEdge;
 #ifndef NSMaxY
 #define NSMaxY CGRectGetMaxY
 #endif
+
+typedef NS_ENUM(NSInteger, NSRectEdge) {
 #ifndef NSMinXEdge
```

Comment:
Is `ifndef` needed?

Reply:
Doing this keeps the same behavior as previously in the removed code